guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -106,20 +106,50 @@ Do not install or update any packages at this stage.
 # vi /etc/yum.repos.d/sat6.repo
 ----
 
-... Change the default `InstallMedia` repository name to `{Project}-{ProjectVersion}`:
+.. Change the default `InstallMedia` repository name to `{Project}-{ProjectVersion}`:
```

Comment:
why this level change? these are (technically) sub-items of the "edit the repo file" section.